src/controllers/user.controller.js

```diff
@@ -0,0 +1,78 @@
+const userService = require('../services/user.service.js');
+
+const get = (_req, res) => {
+  res.sendStatus(200);
```

Comment:
Calling `res.sendStatus(200)` followed by `res.send()` will result in an error because the response is already sent with the status code. Remove `res.sendStatus(200)` and directly use `res.send(userService.getAll())`.